app/database_etl/test_adjustment_handler/create_test_set.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
```

Comment:
maybe we can find another folder to commit this in? we may need this script for later if we want to prototype more test adjustment models but doesn't rly fit with the test adjustment handler

Reply:
added this to shared scripts submodule